packages/swapper/src/api.ts

```diff
@@ -60,6 +60,7 @@ type CommonTradeInput = {
   sendMax: boolean
   sellAssetAccountNumber: number
   wallet?: HDWallet // TODO remove this in a followup PR
```

Comment:
why is this optional? is this comment still valid?

Reply:
fixed